src/modules/quick_api_module.f90

```diff
@@ -535,6 +543,10 @@ subroutine run_quick(self,ierr)
 #if defined CUDA || defined CUDA_MPIV
   ! upload molecular and basis information to gpu
   if(.not.quick_method%opt) call gpu_upload_molspecs(ierr)
+
+#ifdef CEW
+  call gpu_set_cew(quick_cew%use_cew,ierr)
```

Comment:
1. Why do we call `gpu_set_cew()` here when we use the `upload()` interface in line 294? 
2. The function signature does not look right. The implementation in `gpu.cu` takes only 1 argument.
3. The function prototype is missing in `cuda.h`

Reply:
Fixed.